client/src/pages/Home.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { useState, useEffect, useRef } from 'react';
 import '../style/Home.css';
 import NavBar from '../components/NavBar.jsx';
-import MapComponent from '../components/MapComponent'; 
+import MapComponent from '../components/MapComponent';
+import FilterSideBar from '../components/FilterSidebar.jsx';
```

Comment:
Make sure to correct the typo here from `.../components/FilterSidebar,jsx` to `../components/FilterSideBar.jsx` (case sensitivity for the b in Sidebar to be capitalized)